src/activities/components/ActivityList.vue

```diff
@@ -82,16 +82,28 @@
       :offset="100"
       @load="displayMoreActivities"
     >
-      <ActivityItem
-        v-for="activity in displayedActivities"
-        :key="activity.id"
-        v-measure
-        :activity="activity"
-        :place-link="placeLink"
-        @join="$emit('join', arguments[0])"
-        @leave="$emit('leave', arguments[0])"
-        @detail="$emit('detail', arguments[0])"
-      />
+      <template
+        v-for="(day, index) in displayedActivitiesGroupedByDate"
+      >
+        <div
+          v-if="!dense"
+          :key="`day-${index}`"
```

Comment:
Minor: I think `day.date` would make a better `key`, as it's related to the data, not just the position in the array (we live update a lot of stuff via websockets, so the contents can change - although not sure about this particular case).

Reply:
If someone would add two activities - one this year and one on the same date a year into the future, then two keys would be the same:
- Tue, February 9, 2021
- Tue, February 9, 2022

This would result in two :keys being `Tue, February 9` each. And Vue would scream :)

Also, if new data arrives, the computed prop that is being looped over would recalculate and so are the keys of the `v-for`. It is save to do this ;)